pom.xml

```diff
@@ -0,0 +1,70 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.jvm-profiling-tools</groupId>
+    <artifactId>async-profiler</artifactId>
+    <version>1.8.2-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-clean-plugin</artifactId>
+                <version>3.1.0</version>
+                <configuration>
+                    <filesets>
+                        <fileset>
+                            <directory>build</directory>
+                            <directory>target</directory>
+                        </fileset>
+                    </filesets>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <compileSourceRoots>src/api/</compileSourceRoots>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.0</version>
+                <configuration>
+                    <outputDirectory>target/</outputDirectory>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>build_native_binary</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>/usr/bin/make</executable>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>add_native_binary_to_jar</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>/usr/bin/zip</executable>
+                            <arguments>
+                                <argument>-u</argument>
+                                <argument>target/async-profiler-${project.version}.jar</argument>
+                                <argument>build/libasyncProfiler.so</argument>
```

Comment:
`async-profiler` supports [many platforms](https://github.com/jvm-profiling-tools/async-profiler#download) having prebuilt libraries and Maven artifact we are going to publish is expected to be used from any supported platform, so we have to find a way to pack all the binaries.